packages/store/addon/-private/system/references/belongs-to.js

```diff
@@ -6,7 +6,7 @@ import { DEPRECATE_BELONGS_TO_REFERENCE_PUSH } from '@ember-data/private-build-i
 import { assertPolymorphicType } from '@ember-data/store/-debug';
 
 import recordDataFor from '../record-data-for';
-import { peekRecordIdentifier } from '../store/internal-model-factory';
+import { RecordCache } from '../store/internal-model-factory';
```

Comment:
When I see this I think "not an instance of something".  But I see this title case used elsewhere so I assume this was blessed/common convention.

Reply:
We use it fairly consistently for both classes and module scope constants (we also use scream case for module scope constants, we should maybe move to that all the time)